src/Symfony/Component/Notifier/Bridge/Bluesky/Tests/BlueskyTransportTest.php

```diff
@@ -344,28 +344,28 @@ public function testReturnedMessageId()
     public static function sendMessageWithEmbedDataProvider(): iterable
```

Comment:
```suggestion
    public static function sendMessageWithEmbeddedDataProvider(): iterable
```

While at it? Well... is it "sendMessageWithEmbedded data provider" or ""sendMessageWithEmbeddedData provider"?

Reply:
the key in the `$options` array is named `embed`: https://github.com/symfony/symfony/pull/59667/files#diff-cfc4fbfd29606a98e9e11f5db74f5f5dd88a6b6beabf2e0fbc31b66419d47c1fR102